### PR TITLE
Linting to unpollute metadata pull request

### PR DESCRIPTION
**Pull request by andysellick**
* Changes in metadata expand/collapse PR had included linting, making reading the changes difficult. This commit contains all of the linting and none of the changes, so should make that PR more readable once merged

### CODE REVIEW

**Comment by mcgoooo** [comment]
@andysellick looks fine

File: app/assets/javascripts/start-modules.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-//= require govuk/modules
-//= require modules/global-bar
-//= require modules/sticky-element-container
-//= require modules/toggle
-//= require modules/track-click
+// = require govuk/modules
```
Comment (fofr) [inline app/assets/javascripts/start-modules.js]: Presumably this doesn't break the way Rails asset pipeline includes these files?

Comment (andysellick) [inline app/assets/javascripts/start-modules.js]: I don't know about all of them, but toggle definitely still works in the front end.

Comment (mcgoooo) [inline app/assets/javascripts/start-modules.js]: it shouldn't do, it should include everything in modules

**Pull request merged**